content/guides/component-testing/quickstart-svelte.md

```diff
@@ -0,0 +1,116 @@
+---
+title: 'Quickstart: Svelte'
+---
+
+<CtBetaAlert></CtBetaAlert>
```

Comment:
@mike-plummer is this beta or alpha? we should match the changelog

Reply:
@ZachJW34 Can you take a look? I believe Svelte is marked as "Alpha" elsewhere, so it shouldn't be "Beta" here

Reply:
That is the banner atop all framework configurations noting that CT testing is in beta, but each framework can be marked as alpha [here](https://cypress-documentation-git-1070-release-cypress-io.vercel.app/guides/component-testing/component-framework-configuration#Automatic-Configuration-Recommended)